python/grass/temporal/register.py

```diff
@@ -226,7 +226,9 @@ def register_maps_in_space_time_dataset(
                 found = gscript.find_file(element=type, name=mapname)
                 if found["mapset"] is not None and len(found["mapset"]) > 0:
                     map_mapset = found["mapset"]
```

Comment:
This practically says "if mapset was retrieved, use that else use the current one" is that correct? Perhaps add sentences to describe different states.

Reply:
Indeed, that makes me wonder does it make sense? If a map is not found on the current search path, should it then be registered as part of the current mapset? Isn`t a an error more appropriate then?

Reply:
If it is not found by find_file, then it does not exist and that's it, no? ...but I don't know the full context here. A function with clearly defined purpose might be go good step if it is even needed here.

Reply:
Solved in the `build_id_from_search_path`function